Makefile

```diff
@@ -84,6 +84,10 @@ endef
 .musl_target:
 	$(eval CARGO_BUILD_FLAGS += --target ${RUST_TARGET_STATIC})
 
+.clean_libz_sys:
+	${CARGO} clean --target ${RUST_TARGET_STATIC} -p libz-sys
+	${CARGO} clean --target ${RUST_TARGET_STATIC} --release -p libz-sys
```

Comment:
Should we prefix the instrunction with `@` to hide the command print and make it clear

Reply:
Fixed